ui/actions/nuxeo-move-content-to-coldstorage-button.js

```diff
@@ -0,0 +1,156 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import '@polymer/iron-icon/iron-icon.js';
+import '@polymer/iron-icons/iron-icons.js';
+import '@nuxeo/nuxeo-elements/nuxeo-element.js';
+import '@nuxeo/nuxeo-elements/nuxeo-operation.js';
+import '@polymer/paper-button/paper-button.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/polymer/lib/elements/dom-if.js';
+import '@polymer/polymer/lib/elements/dom-repeat.js';
+import { FiltersBehavior } from '../nuxeo-filters-behavior.js';
+import { FormatBehavior } from '../nuxeo-format-behavior.js';
+import '../nuxeo-icons.js';
+import '../widgets/nuxeo-dialog.js';
+import '../widgets/nuxeo-tooltip.js';
+import './nuxeo-action-button-styles.js';
+
+{
+  /**
+   * A button element to move the document's content to the cold storage.
+   *
+   * Example:
+   *
+   *    <nuxeo-move-content-to-coldstorage-button document="[[document]] user="[[user]]"">
+   *    </nuxeo-move-content-to-coldstorage-button>
+   *
+   * @customElement
+   * @appliesMixin mixinBehaviors
+   * @polymerBehavior Nuxeo.FiltersBehavior
+   * @polymerBehavior Nuxeo.FormatBehavior
+   * @memberof Nuxeo
+   */
+  class MoveToColdStorage extends mixinBehaviors([FiltersBehavior, FormatBehavior], Nuxeo.Element) {
```

Comment:
Generically, all the new elements that we introduce to `nuxeo-elements` should follow the same guidelines the migrated ones (please see [here](https://jira.nuxeo.com/browse/ELEMENTS-1024))

Also, we'll need to clarify if Glacier/ColdStorage is, or not, an addon. Because if it is, we may want to move the code to WebUI repository.